### PR TITLE
Added support for comma-separated hosts config parameter

### PR DESCRIPTION
**Pull request by e-kolpakov**
**Description:** Adds support for comma-separated hosts configuration

See motivation and discussion at #117

### CODE REVIEW

**Comment by e-kolpakov** [comment]
@timcharper Please take a look when you have time.

**Comment by e-kolpakov** [comment]
@timcharper Friendly reminder there's some nice-to-have feature waiting for your approval.

File: core/src/main/scala/com/spingo/op_rabbit/ConnectionParams.scala
Comment (timcharper) [inline core/src/main/scala/com/spingo/op_rabbit/ConnectionParams.scala:11]: I'd like to move away from JavaConversions; they're harmful

**Review by timcharper — APPROVED** [review]

**Pull request merged**
